maven-plugin/src/main/java/org/openmrs/maven/plugins/CreateProject.java

```diff
@@ -278,7 +283,7 @@ private void createModule() throws MojoExecutionException {
 
 		if (TYPE_PLATFORM.equals(type)) {
 			platform = wizard.promptForValueIfMissingWithDefault(
-			    "What is the lowest version of the platform (-D%s) you want to support?", platform, "platform", "1.11.6");
+			    "What is the lowest version of the platform (-D%s) you want to support?", platform, "platform", "2.4.0");
```

Comment:
i made `2.4.0` as the default but the attempt was to make it the least version of platform that one can create a platform module for (i will make a commit to fix that).

@dkayiwa platform `2.4.4` and `2.4.5` is very problematic , can we make `2.5.0` the least platform version a platform module can be generated for?